### PR TITLE
Layer descriptions

### PR DESCRIPTION
**Pull request by ZachTRice**
Layer descriptions contained within am "i" button in the sidebar. If no description is available a disabled "i" button is shown. If a description is available, clicking the "i" button expands a modal box. The "i" button and modal box are only available within the desktop view.
Connects to #65
connects to #316

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -382,13 +383,13 @@
       </header>
       <div id="layer-modal-main">
         <div id="layer-categories">
-          
+
         </div>
         <div id="selected-category">
-          
+
```
Comment (Benjaki2) [inline web/index.html]: I'd leave out these spaces if there isn't anything inside the divs `<div></div>`

Comment (ZachTRice) [inline web/index.html]: I agree. This is how it is currently on master, though I should have cleaned it up while I was working in this area. I'm not sure why git is picking this up as a change.

File: web/js/layers/wv.layers.info.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    var loaded = function(custom) {
+        $dialog = wv.ui.getDialog();
+        $dialog
+            .attr("id", "wv-layers-info-dialog")
+            .attr("data-layer", layer.id);
+        renderDescription($dialog);
+
+
+        var names = models.layers.getTitles(layer.id);
```
Comment (Benjaki2) [inline web/js/layers/wv.layers.info.js]: declare vars at top of scope

```
var loaded = function(custom) {
     var names;
```

File: web/index.html (hunk 1 above)
Comment (ZachTRice) [inline web/index.html]: Ah, the differences is that there were spaces after the line return that are no longer present.

File: web/js/layers/wv.layers.info.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        loaded();
+    };
+
+    var loaded = function(custom) {
+        $dialog = wv.ui.getDialog();
+        $dialog
+            .attr("id", "wv-layers-info-dialog")
+            .attr("data-layer", layer.id);
+        renderDescription($dialog);
+
```
Comment (Benjaki2) [inline web/js/layers/wv.layers.info.js]: excess line

File: web/index.html (hunk 1 above)
Comment (Benjaki2) [inline web/index.html]: ahhh makes sense. Thanks for cleaning up those spaces.

**Comment by ZachTRice** [comment]
Thanks, I appreciate the nit picky! 

Couple things I added to that last commit. Removed a `</div>` after the opening `<body>` tag that's been in master. Also, just noticed I had to tweak the css a bit to add back in the `li` / `a` elements to the class selectors since bank.css is setting these and my hover state colors were being over written with the default link color. 

At some point, I'd also like to remove as much element.class selector's as possible. Just to make scoping and compartmentalizing the css a bit easier.

**Comment by Benjaki2** [comment]
Wait till after next release to merge

**Comment by ZachTRice** [comment]
Closing this PR because this feature will be merged into a new branch with search-descriptions. The new combined feature branch will then have a PR into development.

**Pull request closed without merging**